flutter_local_notifications/lib/src/platform_specifics/darwin/notification_details.dart

```diff
@@ -139,4 +140,15 @@ class DarwinNotificationDetails {
   /// This property is only applicable to iOS 15.0 and macOS 12.0 or newer.
   /// https://developer.apple.com/documentation/usernotifications/unnotificationcontent/3747256-interruptionlevel
   final InterruptionLevel? interruptionLevel;
+
+  /// The volume for playing the critical alert sound.
```

Comment:
Can you separate the first sentence so it has its own paragraph? This is so it can follow [this rule](https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph)